src/main/java/frc/robot/RobotPreferences.java

```diff
@@ -227,6 +227,25 @@ public static final class VisionPrefs {
     public static final SN_DoublePreference limelightMountHeight = new SN_DoublePreference(
         "limelightMountHeight", 42);
     public static final SN_DoublePreference limelightMountAngle = new SN_DoublePreference("limelightMountAngle", 35.6);
+
+    // Regression Variables
+    public static final SN_DoublePreference lowHoodMaxDistance = new SN_DoublePreference("lowHoodMaxDistance", 50);
+    public static final SN_DoublePreference midHoodMaxDistance = new SN_DoublePreference("midHoodMaxDistance", 90);
+    public static final SN_DoublePreference highHoodMaxDistance = new SN_DoublePreference("highHoodMaxDistance", 300);
+
+    // Low Hood
+    public static final SN_DoublePreference linearLowA = new SN_DoublePreference("linearLowA", 3050);
+    public static final SN_DoublePreference linearLowB = new SN_DoublePreference("linearLowB", -7.14286);
```

Comment:
variable names should be `regLowA` and `regLowB` to stay consistent with the below names. `linearLowA` is like `quadMidA`. a linear regression model is just as much a regression model as a quadratic or cubic regression model. if you disagree just ignore.